src/time_tracker/migration.clj

```diff
@@ -0,0 +1,10 @@
+(ns time-tracker.migration
+  (:require [ragtime.repl]
+
+            [time-tracker.config :refer [migration-config]]))
+
+(defn migrate-db []
+  (ragtime.repl/migrate migration-config))
+
+(defn rollback-db []
```

Comment:
AFAIK, this will rollback *all* the commits. I think you want to rollback only the last one.